test/lib/ansible_test/_data/completion/remote.txt

```diff
@@ -5,7 +5,7 @@ fedora become=sudo provider=aws arch=x86_64
 freebsd/13.4 python=3.11 python_dir=/usr/local/bin become=su_sudo provider=aws arch=x86_64
 freebsd/14.2 python=3.11 python_dir=/usr/local/bin become=su_sudo provider=aws arch=x86_64
 freebsd python_dir=/usr/local/bin become=su_sudo provider=aws arch=x86_64
-macos/14.3 python=3.11 python_dir=/usr/local/bin become=sudo provider=parallels arch=x86_64
+macos/15.3 python=3.11 python_dir=/usr/local/bin become=sudo provider=parallels arch=x86_64
```

Comment:
The new Python version should be reflected here.

Reply:
Yeah, I haven't yet rebuilt the image with 3.13 but wanted to test it out. I'll update it once the new image is up.

Reply:
Done. Marking as resolved.